app/config/config.yml

```diff
@@ -3,6 +3,11 @@ imports:
     - { resource: security.yml }
     - { resource: services.yml }
 
+# Put parameters here that don't need to change on each machine where the app is deployed
+# http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    locale: en
```

Comment:
That makes sense, I like the idea of moving that as well. 
One minor concern to think about: Do you think that harms the getting-started experience for new users with a different environment set-up?